balancer/frontend-v2
# Avalanche promo tile

# Description

1. Added a new promo banner for the Avalanche launch
2. Added a 'NEW' badge in the network selector for Avalanche

## Type of change

- [x] Other: UI update

## How should this be tested?

Check that the changes looks as follows:

1. New promo banner for the Avalanche launch

<img width="1010" alt="CleanShot 2023-07-26 at 23 34 37@2x" src="https://github.com/balancer/frontend-v2/assets/1121139/77b88884-28e2-405f-90de-37e896e1e23e">

2. 'NEW' badge in the network selector for Avalanche

<img width="253" alt="CleanShot 2023-07-26 at 23 39 24@2x" src="https://github.com/balancer/frontend-v2/assets/1121139/3f38ba44-d926-4a28-a269-c45029bef95e">

## Checklist:
